sevenseconds/config/vpc.py

```diff
@@ -781,3 +876,20 @@ def cleanup_vpc(account: AccountData, region: str):
     with ActionOnExit('Delete Elastic IPs..'):
         for eip in ec2c.describe_addresses()['Addresses']:
             ec2c.release_address(AllocationId=eip['AllocationId'])
+
+
+def get_sg(name: str, desc: str, sgs: list) -> object:
+    for sg in sgs:
+        if sg.group_name == name and sg.description == desc:
+            return sg
+
+
+def allow_https_all(permissions: list) -> bool:
+    for permission in permissions:
+        if (
+            permission['IpProtocol'] == 'tcp'
+            and permission['FromPort'] == 443
+            and permission['ToPort'] == 443
+            and {'CidrIp': '0.0.0.0/0'} in permission['IpRanges']
```

Comment:
I'd still use the VPC IP address here, considering weird edge cases like peered VPCs and VPNs.